### PR TITLE
alex stapleton, 4 u

### PR DESCRIPTION
**Pull request by beetai**
we gotta delete that loading redux stuff :(

### CODE REVIEW

File: frontend/src/components/Results.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const Results = (props: Props) => {
   return (
     <Grid container spacing={3}>
       {searchResults.map((item) => (
-        <Grid item xs={4}>
+        <Grid item xs={4} key={item.courseId}>
```
Comment (DSep) [inline frontend/src/components/Results.tsx:37]: thx

**Review by DSep — APPROVED** [review]
👌
was fun
we shall do more
thanks for making it good

File: frontend/src/components/Results.tsx (hunk 1 above)
Comment (yqj2k) [reply frontend/src/components/Results.tsx:37]: me too

File: frontend/src/pages/Browse.tsx [hunk 2]
```diff
@@ -35,6 +36,9 @@ const Browse = (props: any) => {
         <Grid item xs />
       </Grid>
       <Search />
+      {isSearchLoading ?
+        <CircularProgress /> :
```
Comment (yqj2k) [inline frontend/src/pages/Browse.tsx:40]: 👀 loading boi

File: frontend/src/redux/reducers/Search.tsx [hunk 3]
```diff
@@ -49,15 +49,27 @@ const sampleCourse1: Course = {
...
-      ];
+      return action.offerings;
+    case SUBMIT_SEARCH_FAILURE:
+      return [];
     default:
       return store;
   }
 };
+
+export const isSearchLoading = (isLoading: boolean = false, action: any) => {
```
Comment (yqj2k) [inline frontend/src/redux/reducers/Search.tsx:64]: something i am unable to achieve: reducers 😆 

good additions

**Review by yqj2k — APPROVED** [review]
LGTM. stapleton would be proud

**Pull request merged**